### PR TITLE
Use apt repos to build the images

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**

### CODE REVIEW

**Comment by leandro-lucarella-sociomantic** [comment]
@mihails-strasuns it might be a good time to see if we want to bump any other version for the v2 images (dmd-transitional?)

**Comment by leandro-lucarella-sociomantic** [comment]
Arrrggg, damn, git package version also depends on the distro name, but with the number :S

I'll fix tomorrow. I'll remove the git version commit for now to see if the rest works.

**Review by mihails-strasuns-sociomantic — DISMISSED** [review]
Removing things, yay!

**Comment by leandro-lucarella-sociomantic** [comment]
Still having some problems with packages not being available for trusty or xenial :unamused:

**Comment by leandro-lucarella-sociomantic** [comment]
Also Git version is a mess, so I'm looking to add a simple tool to get the latest version of a package that starts with some string. This will allow us to specify versions using like `1.6` and then cachalot will install the latest 1.6.x. Which I think it makes sense, it will be quite annoying to have to bump patch versions manually.

**Comment by mihails-strasuns-sociomantic** [comment]
Also btw you have pinged non `-sociomantic` mihails first time (this is another my private account ;))

**Comment by leandro-lucarella-sociomantic** [comment]
> Also btw you have pinged non -sociomantic mihails first time (this is another my private account ;))

Woops! Sorry!

**Comment by nemanja-boric-sociomantic** [comment]
But it's much more convenient :-)

**Comment by leandro-lucarella-sociomantic** [comment]
Right on! :green_heart:

**Comment by leandro-lucarella-sociomantic** [comment]
This is ready for a final review @mihails-strasuns-sociomantic @nemanja-boric-sociomantic

File: docker/base [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xeu
 
+# Get utility stuff
+. ./util.sh
```
Comment (mihails-strasuns-sociomantic) [inline docker/base]: Minor nitpick: is there any semantical difference between `. ./util.sh` and `source ./util.sh`? I usually prefer former as more obvious.

**Review by mihails-strasuns-sociomantic — APPROVED** [review]
I haven't checked python script in any details but everything else looks good.

File: docker/base (hunk 1 above)
Comment (leandro-lucarella-sociomantic) [reply docker/base]: `source` is a bashism, it doesn't work in plain `sh`.

Comment (mihails-strasuns-sociomantic) [reply docker/base]: :grumpy:

**Comment by leandro-lucarella-sociomantic** [comment]
OK, merging. @nemanja-boric-sociomantic can make comments to be addressed in other PRs if needed.

**Pull request merged**